platform/posix/transport/utest/openssl_utest.c

```diff
@@ -849,8 +849,9 @@ void test_Openssl_Recv_All_Bytes_Received_Successfully( void )
     TEST_ASSERT_EQUAL( BYTES_TO_RECV, bytesReceived );
 
     /* Pending data from last read. */
+    SSL_pending_ExpectAnyArgsAndReturn( BYTES_TO_RECV );
     SSL_read_ExpectAnyArgsAndReturn( BYTES_TO_RECV );
```

Comment:
Is there a reason for returning `BYTES_TO_RECV` from `SSL_read` when only a single byte is requested in the test?

Reply:
I had forgotten to update it